mobile/docs/naming/README.md

```diff
@@ -0,0 +1,51 @@
+# Wolox Mobile Convention Naming
+
+In Wolox we use conventions for naming projects and identifiers, as `Bundle ID` for `iOS` projects and `Application ID` for `Android`.
```

Comment:
`At Wolox` (I might be wrong, but`in` is more best used as a physical position).